pom.xml

```diff
@@ -114,13 +116,16 @@
       </properties>
     </profile>
     <profile>
-        <id>spark-3.2</id>
-        <properties>
-          <scala.version>${spark32.scala}</scala.version>
-          <spark.version>${spark32.version}</spark.version>
-          <delta.version>${delta1.version}</delta.version>
-          <sparkbundle.version>${spark32bundle.version}</sparkbundle.version>
-        </properties>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
Why need this one?

Reply:
We either remove all `<activeByDefault>true</activeByDefault>` under `<id>spark-3.2</id>`, either add to them all. Maybe we just remove all of them?

Reply:
In the old logical, it will activate -Pspark at default, if we remove -Pspark, I think we can add to them all.